docs/src/concepts/smartweave.md

```diff
@@ -0,0 +1,26 @@
+# SmartWeave
+
+## What is SmartWeave?
+
+The SmartWeave protocol is the smartcontract protocol for Arweave. The major difference from other blockchains - namely Ethereum, one of the leaders in smartcontract ecosystems - is Arweave nodes do not validate smartcontracts, but rather the user themselves when they interact with the contract, thru the use of "lazy-evaluation". SmartWeave contracts currently support JavaScript, TypeScript, Rust, Go, AssemblyScript, and WASM (WebAssembly).
+
+## Why is SmartWeave important?
+
+SmartWeave is an important part of the Arweave ecosystem - it allows transfer of ownership of Atomic assets (NFTs) and updatable ledger-like record keeping.
+
+## How does SmartWeave Work?
+
+SmartWeave contracts, at their core, are built from an initial contract state, with edits and additions using transaction tags. SmartWeave SDK's (such as Warp {previously RedStone} and the legacy SmartWeave SDK) are used to query for these transactions to build contract state locally, adding or subtracting from the contract state with each transaction in order from oldest to newest. The resulting state is the current contract state, which the SDK on the client side can use to calculate user balances, contract owners, and other contract specific details. Once the Dapp has a validated contract state they can build an interaction for the user to deploy to the chain, which upon mining or indexing on a gateway (see gateway Core Concept) will be included the next time someone builds the contract state. 
```

Comment:
Instead of `once the Dapp` perhaps `once the caller`?

dApp is a little bit of a poisoned term, there was such an excitement over building decentralized applications, people called them dApps and hardly any were actually decentralized.  Permaweb apps are decentralized by default so it sort of weakens them to call them `dApps` even though it's a common term. YMMV

Reply:
added